app/views/devices_guidance/how_to_order_for_social_care.html.erb

```diff
@@ -0,0 +1,215 @@
+      Tell us if your allocation is wrong
+    </h4>
+    <p class="govuk-body">
+      If your estimated allocation is wrong, <%= govuk_link_to 'contact us', support_ticket_path %> and let us know how
+      many laptops you currently need for eligible care leavers. Do not include any personal data, such as their names or
+      dates of birth.
+    </p>
+
+    <p class="govuk-body">
+      We cannot guarantee that we’ll be able to meet all requests for more laptops. We encourage you to order your
+      current allocation anyway so that you can begin distributing devices. We’ll contact you once we’ve reviewed your request.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      2. Check what care leavers need
+    </h3>
+
+    <p class="govuk-body">
+      We recommend working with your care leavers and IT team as soon as possible to find out the following information.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Laptops
+    </h4>
+
+    <p class="govuk-body">To order laptops, you need to find out which:</p>
+
+    <ol class="govuk-list govuk-list--number">
+      <li>
+        Eligible care leavers do not have access to a laptop or tablet.
+      </li>
+      <li>
+        <%= govuk_link_to "Type of laptop and IT settings are most suitable for each care leaver", '/devices/device-specification' %>
+        - if they need Google Chromebooks, you should ask for their educational setting’s domain for
+        <span class="app-no-wrap">Google Workspace for Education Fundamentals</span>
+        <span class="app-no-wrap">(formerly G Suite for Education)</span>, and recovery email address. We’ll ask you for
+        these after you place your order.
+      </li>
+    </ol>
+
+    <p class="govuk-body">
+      See <%= govuk_link_to 'step 6', '#step-6' %> to find out about safeguarding responsibilities, laptop ownership and providing IT support.
+    </p>
+
+    <h4 class="govuk-heading-s">
+      Internet access
+    </h4>
+
+    <p class="govuk-body">
+      To request internet access, you need to find out which eligible care leavers do not have fixed broadband.
+    </p>
+
+    <p class="govuk-body">
+      We cannot guarantee that we’ll be able to meet all requests for more laptops. We encourage you to order your
+      current allocation anyway so that you can begin distributing devices. We’ll contact you once we’ve reviewed your
+      request.
+    </p>
+
+    <h3 class="govuk-heading-m">
+      3. Complete and submit your contact details form
+    </h3>
+
+    <p class="govuk-body">
+      We need some information from you to set up your Get help with technology service account, even if you’ve
+      used the service before. You’ll place your orders using this account.
+    </p>
+
+    <p class="govuk-body">
+      If you’ve been nominated to order for your LA, you’ll receive an email with a link to an online form. You need to use that form to tell us:
+    </p>
+
+    <ol class="govuk-list govuk-list--number">
```

Comment:
Please use an unordered list:

- your full name
- your local authority’s name and 3-digit code
- the delivery address we should send your laptops to (all devices will be delivered to the same location at your local authority)
- the person we should contact when we deliver your laptops

Reply:
Please remove full stops (each line should be lower case, no full stop). Thanks